core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -365,18 +365,18 @@ trait WorkspaceComponent {
         updateLastModified(workspaceId)
     }
 
-    def getAuthorizedRealms(workspaceIds: Seq[String], user: RawlsUserRef): ReadAction[Seq[Option[RawlsGroupRef]]] = {
+    def getAuthorizedRealms(workspaceIds: Seq[String], user: RawlsUserRef): ReadAction[Seq[Option[RawlsRealmRef]]] = {
       val realmQuery = for {
         workspace <- workspaceQuery if workspace.id.inSetBind(workspaceIds.map(UUID.fromString))
       } yield workspace.realmGroupName
 
       realmQuery.result flatMap { allRealms =>
         val flatRealms = allRealms.flatten.toSet
         DBIO.sequence(flatRealms.toSeq.map { realm =>
-          val groupRef = RawlsGroupRef(RawlsGroupName(realm))
-          rawlsGroupQuery.loadGroupIfMember(groupRef, user) flatMap {
+          val realmRef = RawlsRealmRef(RawlsGroupName(realm))
+          rawlsGroupQuery.loadGroupIfMember(realmRef, user) flatMap {
```

Comment:
No, there's no need to do a `sequence()` because you're getting all groups at once.  Then you can put the resulting `Seq[RawlsRealmRef]` inside a `DBIO.successful()`